docs/package.json

```diff
@@ -24,7 +24,7 @@
     "start-server-and-test": "^1.12.6"
   },
   "dependencies": {
-    "@docsearch/react": "^1.0.0-alpha.27"
+    "@docsearch/react": "^3.0.0-alpha.41"
```

Comment:
this will probably break! I got an email about this update with new permissions / API key, I can share if you're able to take a look.